### PR TITLE
bump telemetry to 0.5.2

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
Telemetry 0.5.0 is broken, we need this patch in 2.25.x.

**What type of PR is this?**
/kind bug
/kind regression

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
Fix telemetry agent container images not starting up.
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: charts/telemetry/.helmignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test/
```
Comment (archups) [inline charts/telemetry/.helmignore]: Minor fix: End of line is missing.

**Review by archups — CHANGES_REQUESTED** [review]

File: charts/telemetry/.helmignore (hunk 1 above)
Comment (xrstf) [reply charts/telemetry/.helmignore]: fixed.

**Review by archups — APPROVED** [review]
/lgtm

**Comment by archups** [comment]
@xrstf your approval is required to merge it. Thanks!

**Pull request merged**